src/CommunityToolkit.Maui.Core/Views/DrawingView/IDrawingView.shared.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.ObjectModel;
+using System.Windows.Input;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// The DrawingView allows you to draw one or multiple lines on a canvas
+/// </summary>
+public interface IDrawingView : IView
+{
+	/// <summary>
+	/// Event occurred when drawing line completed
+	/// </summary>
+	event EventHandler<DrawingLineCompletedEventArgs> DrawingLineCompleted;
+
+	/// <summary>
+	/// The <see cref="Color"/> that is used by default to draw a line on the <see cref="IDrawingView"/>. This is a bindable property.
+	/// </summary>
+	Color LineColor { get; }
+
+	/// <summary>
+	/// The width that is used by default to draw a line on the <see cref="IDrawingView"/>. This is a bindable property.
+	/// </summary>
+	float LineWidth { get; }
+
+	/// <summary>
+	/// This command is invoked whenever the drawing of a line on <see cref="IDrawingView"/> has completed.
+	/// Note that this is fired after the tap or click is lifted. When <see cref="MultiLineMode"/> is enabled
+	/// this command is fired multiple times.
+	/// This is a bindable property.
+	/// </summary>
+	ICommand? DrawingLineCompletedCommand { get; }
```

Comment:
So, `ICommand` is very MVVM, on `Core` we should be agnostic about patterns

Reply:
Agree, it can be removed. We have OnDrawingCompleted method.